src/utils/interface.js

```diff
@@ -1,3 +1,6 @@
+
+const KEY_CODES = { q: 81, v: 86 };
```

Comment:
wait sorry, do you want `Q` and `V` capitalized? I was confused b/c I wasn't sure if typing shift-q and shift-v was a different key code, and that might be confusing.